app/views/machines/show.html.erb

```diff
@@ -1 +1,9 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+<section class="snack-list">
+<% @machine.snacks.each do |snack| %>
+  <p><%= snack.name %>: $<%=  snack.price %></p>
+<% end %>
+</section>
+<section class="snack-stats">
+  <p>Average snack price: $<%= @machine.average_snack_price.to_f.round(2) %></p>
```

Comment:
you can look into the action view helper, `number_to_currenct` for formatting this output.